src/components/LinkTab/LinkTab.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Tab from "@mui/material/Tab";
+
+const LinkTab = (props) => (
+  <Tab component={Link} to={props.pathname} {...props} />
```

Comment:
Here is a small amount of props, we can describe all of them here and define their types